module1.js

```diff
@@ -38,7 +45,10 @@ function question2(number1, number2) {
   Return a string "odd" if the number is odd, or a string "even" if the number is even.
 */
 
-function question3(number) {
+function checksIfAnInputIntegerIsOddNumberOrEven(5,7) {
+  if (odd) { return a string odd;
+    if (even){return a string even;
```

Comment:
You never closed the first `if` block with a `}`, so even if this correctly checked whether the input was even it would never be reached. And if the first block was closed it would make more sense for this to be an `else` than another `if` - assuming we only deal with integers, if it's not odd it must be even.